ksqldb-parser/src/test/java/io/confluent/ksql/util/GrammarTokenExporterTest.java

```diff
@@ -41,7 +41,7 @@ public class GrammarTokenExporterTest {
       "CATALOG", "PROPERTIES", "BEGINNING", "UNSET", "RUN", "SCRIPT", "DECIMAL",
       "KEY", "CONNECTOR", "CONNECTORS", "SINK", "SOURCE", "NAMESPACE", "MATERIALIZED",
       "VIEW", "PRIMARY", "REPLACE", "ASSERT", "ADD", "ALTER", "VARIABLES", "PLUGINS", "HEADERS",
-      "HEADER", "SYSTEM", "IF", "EQ", "NEQ", "LT", "LTE", "GT", "GTE", "PLUS", "MINUS", "ASTERISK",
+      "HEADER", "SYSTEM", "TIMEOUT", "IF", "EQ", "NEQ", "LT", "LTE", "GT", "GTE", "PLUS", "MINUS", "ASTERISK",
```

Comment:
What's our procedure for adding new reserved words?  I imagine we do want to note that somewhere.

Reply:
`TIMEOUT` is a non-reserved word. This is just a list of all the grammar tokens.